lib/internal/streams/from.js

```diff
@@ -2,8 +2,9 @@
 
 const {
   PromisePrototypeThen,
+  PromisePrototypeResovle,
```

Comment:
Isn't it should be `PromisePrototypeResolve` instead of `PromisePrototypeResovle` (typo alert)?

Reply:
fixed